src/System.Drawing.Primitives/src/System/Drawing/KnownColor.cs

```diff
@@ -7,7 +7,7 @@
 namespace System.Drawing
 {
     [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
-    internal enum KnownColor
+    public  enum KnownColor
```

Comment:
Adding this as a public type in System.Drawing.Primitives is going to cause issues with netstandard. In particular currently the NETStandard.Library contains a shim for System.Drawing.Primitives so at a minimum we need to bump the minor version of System.Drawing.Primitives to ensure folks targeting .NET Core will get the later version.